src/components/Tooltip/Tooltip.tsx

```diff
@@ -33,6 +35,7 @@ export function Tooltip({
   children,
   content,
   light,
+  notRetain = true,
```

Comment:
We shouldn't set boolean props to default as `true`. `<Component notRetain={false} />` isn't the best API. Its better to assume `false` as default and to enable something by doing `<Component retain />`. A `boolean` prop is "binary" in that you either passed the prop without a value or you didn't pass the prop at all.